src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -1978,6 +2096,9 @@ private void DoMembraneChangeAction(MicrobeEditorAction action)
         CalculateEnergyBalanceWithOrganellesAndMembraneType(
             editedMicrobeOrganelles.Organelles, Membrane, targetPatch);
         gui.SetMembraneTooltips(Membrane);
+
+        previewMicrobe.Membrane.Type = membrane;
```

Comment:
This will crash if there is no previewMicrobe when this action is redone/undone. Need to check against null here.